mailbox/api/src/main/java/org/apache/james/mailbox/AttachmentManager.java

```diff
@@ -19,14 +19,12 @@
 
```

Comment:
Please keep these methods ;-)

Even if you think they are not useful, they are:
 - **getOwnerMessageIds** is re-inserted next commit.
 - **storeAttachmentsForMessage** is the method that SHOULD be used. (I even wonder if *storeAttachmentsForMessage* alone makes sens)